service/payment.go

```diff
@@ -244,7 +244,7 @@ func getTotalAmount(costs *[]models.CostResourceRest) (string, error) {
 
 func getCosts(resource string) (*models.CostsRest, ResponseType, error) {
 
-	resourceReq, err := http.NewRequest("GET", resource, nil)
+	resourceReq, err := http.NewRequest("GET", resource+"/payment", nil)
```

Comment:
Is it correct for the Payments API to be defining the path for another unknown service?
It seems strange to me to be adding `/payment` here.